DESCRIPTION

```diff
@@ -24,16 +24,20 @@ Description: Easily analyze relational data from the 2016 federal
     This package contains data about candidates, commitees, and a
     variety of different financial expenditures. 
 License: CC0
-Depends: R (>= 2.10)
-Suggests: 
+Depends: 
+    R (>= 2.10),
+    dplyr,
+    readr
+Suggests:
     testthat (>= 2.1.0),
+    scales,
     knitr,
     rmarkdown,
-    dplyr,
-    ggplot2,
-    tidyr,
-    scales,
-    ggrepel
+    fs,
+    lubridate,
+    usethis,
+    tidyverse,
```

Comment:
We can't have `tidyverse` as a dependency. If you are using in the vignette, can you do what you want without `tidyverse`?